components/Course/feature-flag-example/page.tsx

```diff
@@ -2,11 +2,11 @@ import Content from "./_client";
 import { getServerAuthSession } from "@/server/auth";
 
 export const metadata = {
-  title: "This is a feature flag example",
```

Comment:
This whole file shouldn't have been touched. Move your logic over to `(app)/courses`. Then your page will be found on `/courses`. The feature flag was something that could be copy and pasted as a template rather than changed.